### PR TITLE
More small screen responsiveness

### LINKED ISSUE #23: Non Breaking Cutting Tools
The line of cutting tools is non-breaking, causing the whole app to effectively have a minimum width. 
In case of smaller screens, we could

- Hide less important buttons in a …-menu
- Reduce the buttons to their icons

### PR DESCRIPTION
**Pull request by Arnei**
Makes the editors appearance on narrow screens a little more palatable. This PR is not intended to make the editor fully usable in portrait mode, but rather help users with narrow screens and/or high zoom levels to get a little more out of their screen space.

Only touches the header, main menu, cutting view and metadata view for now.

Resolves #23, maybe?

### CODE REVIEW

File: src/main/Header.tsx [hunk 1]
```diff
@@ -112,6 +124,7 @@ const Logo: React.FC = () => {
 
   const { t } = useTranslation();
   const { scheme } = useColorScheme();
+  const { width } = useWindowDimensions();
```
Comment (geichelberger) [inline src/main/Header.tsx:127]: Do we need this? Else this line and the useWindowDimensions() import can be removed.

**Review by geichelberger — CHANGES_REQUESTED** [review]
LGTM, except for the not-used value.

File: src/main/Header.tsx (hunk 1 above)
Comment (Arnei) [reply src/main/Header.tsx]: Removed it, thanks for catching that.

**Review by geichelberger — APPROVED** [review]
LGTM

**Pull request merged**